CHANGELOG.md

```diff
@@ -1,4 +1,88 @@
-## 6.17.0 (Unreleased)
+## 6.18.0 (Unreleased)
+
+FEATURES:
+* **New Data Source:** `google_compute_instance_template_iam_policy` ([#20954](https://github.com/hashicorp/terraform-provider-google/pull/20954))
+* **New Data Source:** `google_kms_key_handles` ([#20985](https://github.com/hashicorp/terraform-provider-google/pull/20985))
+* **New Data Source:** `google_organizations` ([#20965](https://github.com/hashicorp/terraform-provider-google/pull/20965))
+* **New Data Source:** `google_parameter_manager_parameter` ([#20953](https://github.com/hashicorp/terraform-provider-google/pull/20953))
+* **New Data Source:** `google_parameter_manager_regional_parameters` ([#20958](https://github.com/hashicorp/terraform-provider-google/pull/20958))
+* **New Resource:** `google_apihub_api_hub_instance` ([#20948](https://github.com/hashicorp/terraform-provider-google/pull/20948))
+* **New Resource:** `google_chronicle_retrohunt` ([#20962](https://github.com/hashicorp/terraform-provider-google/pull/20962))
+* **New Resource:** `google_colab_runtime` ([#20940](https://github.com/hashicorp/terraform-provider-google/pull/20940))
+* **New Resource:** `google_colab_runtime_template_iam_binding` ([#20963](https://github.com/hashicorp/terraform-provider-google/pull/20963))
+* **New Resource:** `google_colab_runtime_template_iam_member` ([#20963](https://github.com/hashicorp/terraform-provider-google/pull/20963))
+* **New Resource:** `google_colab_runtime_template_iam_policy` ([#20963](https://github.com/hashicorp/terraform-provider-google/pull/20963))
+* **New Resource:** `google_compute_instance_template_iam_binding` ([#20954](https://github.com/hashicorp/terraform-provider-google/pull/20954))
+* **New Resource:** `google_compute_instance_template_iam_member` ([#20954](https://github.com/hashicorp/terraform-provider-google/pull/20954))
+* **New Resource:** `google_compute_instance_template_iam_policy` ([#20954](https://github.com/hashicorp/terraform-provider-google/pull/20954))
+* **New Resource:** `google_gemini_code_repository_index` (GA) ([#20941](https://github.com/hashicorp/terraform-provider-google/pull/20941))
```

Comment:
```suggestion
* **New Resource:** `google_gemini_code_repository_index` ([#20941](https://github.com/hashicorp/terraform-provider-google/pull/20941))
```

Similar for the rest of these new resources

Reply:
Hmm- I'd prefer we keep the marker personally, these are promotions